auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManagerException.java

```diff
@@ -15,4 +15,14 @@
     CredentialsManagerException(String message) {
         super(message);
     }
+
+    /**
+     * Returns true when this Android device doesn't support the cryptographic algorithms used
+     * to handle encryption and decryption, false otherwise.
+     *
+     * @return whether this device is compatible with {@link SecureCredentialsManager} or not.
+     */
+    public boolean isDeviceIncompatible() {
```

Comment:
Why have a method that checks for the negative outcome? Why not `isDeviceCompatible` ?

Reply:
Thought about that but discarded it. The thing is this is already an exception (something that went wrong). The "catastrophic" reason of the error could be a `DeviceIncompatibleException`, thus the name of the method.

It wouldn't make sense to receive this exception and check "is device compatible" IMO. Maybe we could rename this method (not the exception class) to `isDeviceNotCompatible`. Thoughts?